File: Piko/Sources/SFInput.m
//
//  SFInput.m
//
//  Copyright (c) 2014 Satoshi Muraki. All rights reserved.
//

#import "SFInput.h"

@implementation SFInput

- (instancetype)initWithName:(NSString *)name
{
    self = [super init];
    if (self != nil) {
        _name = [name copy];
    }
    return self;
}

@end

File: Piko/Sources/SFInput.swift
//
//  SFInput.swift
//
//  Copyright (c) 2014 Satoshi Muraki. All rights reserved.
//

import Foundation

@objc class SFInput: NSObject {
    let name: String
    weak var owner: SFNode?
    weak var source: SFOutput?

    init(name: String) {
        self.name = name
    }
}
